style.css

```diff
@@ -48,11 +48,62 @@ body {
   }
 }
 
-.material-icon {
+.menu-icon {
   width: 24px;
   height: auto;
 }
 
+.hamburger {
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  border: #fff;
+
+  /* background: white; */
+  cursor: pointer;
+  color: #6070ff;
+}
+
+.close-icon {
+  display: none;
+  background: #6070ff;
+  color: #fff;
+}
```

Comment:
To stick with the original design, please make sure that your closing icon looks like the one displayed in Figma 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/201728492-3b01c876-773f-4104-9615-0b233e6c6256.png)  | ![image](https://user-images.githubusercontent.com/42799579/201728461-eb5efbdf-ed80-41d6-8fc1-3877b6b120b3.png)  |

Kindly notice that your button has a darker square around it 😄.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.